engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -3187,6 +3198,18 @@ struct spirit_bomb_t : public demon_hunter_spell_t
         td(s->target)->debuffs.frailty->trigger();
       }
     }
+
+    double composite_da_multiplier( const action_state_t* s ) const override
+    {
+      double m = demon_hunter_action_t::composite_da_multiplier( s );
```

Comment:
I notice here this call is also slightly wrong. Outside of funny cases, generally, we just want to call the direct parent class, whichever is referenced in the constructor. 

```
    spirit_bomb_damage_t( util::string_view name, demon_hunter_t* p )
      : demon_hunter_spell_t( name, p, p->find_spell( 247455 ) )
```

In this case all the entry calls should just be demon_hunter_spell_t::

Reply:
This was just me using the "generate override" and not noticing it was referencing the top level class instead of the parent.